src/index.js

```diff
@@ -1,182 +1,194 @@
-const dicomParser = require('dicom-parser')
```

Comment:
About this file entire change.
Its not the ideal having more changes than necessary (it also applies to this case: i.e lint), but until there is a quality code strategy for linting I strongly recommend during review process to ignoring if possible formatting changes.

Reply:
I'm ignoring formatting changes, but the change below is also a re-ordering change.  If you want to setup a linting strategy, then run that and commit the updates then I would be happier with having you re-order things.